### DISCUSSION TITLE
Monte Carlo Tree Search o1 data generation explained simply

### DISCUSSION THREAD

**Discussion by DataBassGit** [daveshap/Raspberry#34] [Research]
https://youtu.be/nO6sDk6vO0g?si=eIFNX8J2iyopJuGU&t=1141

Will edit this later, but timestamped for convenience.

**Comment by tinycrops** [comment]
hu-po has an expose on it as well https://github.com/hu-po/docs/blob/main/2024.09.13.strawberry/main.md
kaggle is running a MCTS competition too! https://www.kaggle.com/competitions/um-game-playing-strength-of-mcts-variants/overview

**Comment by russellballestrini** [comment]
2023:

![GXkAZoobwAAVFOb](https://github.com/user-attachments/assets/7dbf8346-1433-48e0-a93e-7e1d86cd330c)

2024:
![Screenshot_20240916_085410_Firefox](https://github.com/user-attachments/assets/883c0c70-4392-48d2-84aa-5200c17fdcec)

I highly suspect gpt-o1 is a gpt-4/4o model with different training regiment.
